site-assets/main.css

```diff
@@ -214,6 +214,8 @@ header nav a:not(:last-child) {
 .credit a {
 	margin: 5px;
 	white-space: nowrap;
+	line-height: 50px;
+    display: inline-block;
```

Comment:
Just a nitpick to adjust the indentation for line 218

Reply:
fixed